workloads/hyper-scale/common.sh

```diff
@@ -51,6 +50,20 @@ setup(){
     rosa verify permissions
 }
 
+pre_flight_checks(){
+    echo "Pre flight checks started"
+    export MULTI_AZ=$(rosa describe cluster -c $MGMT_CLUSTER_NAME -o json | jq -r [.multi_az] | jq -r .[])
+    export COMPUTE_NODE=$(rosa describe cluster -c $MGMT_CLUSTER_NAME -o json | jq -r [.nodes.compute] | jq -r .[])
+
+    if [ "${MULTI_AZ}" == "true" ] && [ "${COMPUTE_NODE}"%3 == 0]; then
```

Comment:
Why do we need the compute nodes to be a multiple of 3 in this check? If we deploy rosa/osd cluster with multi-az the cli will only let us give a multiple of 3 for worker node count - so I guess this point is moot.